### ISSUE TITLE
Web Wallet copy suggestion: Pinning extension 

### ISSUE THREAD

**Issue by GinaAbrams** [blockstack/blockstack.org#1180]
We should add that users need to **pin** the extension and how here. https://www.hiro.so/wallet/install-web

**Comment by markmhx** [comment]
> I think it is a blocker to accessing apps

Mind elaborating on the issue here? Has a user or developer reported not being able to find the extension post-installation?

Note that we may want to provide this guidance on the installation confirmation within the Stacks Wallet itself, since we have a similar graphic already (just sans pinning instructions):

![image](https://user-images.githubusercontent.com/28991/106755807-d8fe3800-662e-11eb-8b56-886ad0718b0c.png)

**Comment by GinaAbrams** [comment]
I personally tried using Boom app with this extension, it did not work without first pinning the extension. Does that help?

**Comment by markmhx** [comment]
Yep that's helpful, though a bit surprising. I'll look into this behavior.
